The value of the DEBUG variable is append to CXXFLAGS, thereby
overwriting the optimization level set in the CXXFLAGS environment
variable (-Os on alpine by default). Since we prefer to use our
optimization level DEBUG needs to be unset.

diff --git a/client/Makefile b/client/Makefile
index f977eea..26ffd67 100644
--- a/client/Makefile
+++ b/client/Makefile
@@ -31,7 +31,7 @@ else
 endif
 
 # Simplify building debuggable executables 'make DEBUG=-g STRIP=echo'
-DEBUG=-O3
+DEBUG=
 
 
 CXXFLAGS += $(ADD_CFLAGS) -std=c++0x -Wall -Wextra -Wpedantic -Wno-unused-function $(DEBUG) -DHAS_FLAC -DHAS_OGG -DVERSION=\"$(VERSION)\" -I. -I.. -I../common

